docs/README.md

```diff
@@ -211,6 +237,3 @@ Note:
 
 - [Code - GNU AFFERO GENERAL PUBLIC LICENSE](../LICENSE)
 - [Creative assets -- Attribution-NonCommercial-NoDerivatives 4.0 International](../app/assets/LICENSE.txt)
-
-<!-- Google Analytics -->
-<img src='https://ga-beacon.appspot.com/UA-102304388-1/gitcoinco/web' style='width:1px; height:1px;' >
```

Comment:
why was this removed ?

Reply:
the image is no longer available and shows a dead link so please find and alternative but till then i removed it